lib/java/test/org/apache/thrift/transport/TestAutoExpandingBufferWriteTransport.java

```diff
@@ -19,14 +19,16 @@
 package org.apache.thrift.transport;
 
 import java.nio.ByteBuffer;
+
+import org.apache.thrift.TConfiguration;
 import org.junit.Test;
 import static org.junit.Assert.*;
 
 public class TestAutoExpandingBufferWriteTransport {
 
   @Test
   public void testIt() throws Exception {
-    AutoExpandingBufferWriteTransport t = new AutoExpandingBufferWriteTransport(1, 0);
+    AutoExpandingBufferWriteTransport t = new AutoExpandingBufferWriteTransport(new TConfiguration(), 1, 0);
```

Comment:
Nut sure but .... wouldnt it be better to instantiate TConfiguration() only once and use it through the whole test case or at least inside the method?